examples/plugins/todo-example/admin/src/index.jsx

```diff
@@ -0,0 +1,42 @@
+import { prefixPluginTranslations } from './utils/prefixPluginTranslations';
+import TodoCard from './components/TodoCard';
+
+const name = 'todo-example';
+
+/** @type import('@strapi/strapi/admin').PluginDefinition */
+export default {
+  register(app) {
+    app.registerPlugin({
+      id: 'todo-example',
+      name,
+    });
+  },
+
+  bootstrap(app) {
+    app.getPlugin('content-manager').injectComponent('editView', 'right-links', {
+      name: 'my-plugin-my-compo',
+      Component: TodoCard,
+    });
+  },
```

Comment:
![CleanShot 2024-09-09 at 14 24 19@2x](https://github.com/user-attachments/assets/1cdc40b7-6618-475d-978b-57893c40b668)

The UI looks a bit weird. I think it would look much nicer using the `contentManagerPluginApis.addEditViewSidePanel` so that it gets its own card. Or if the goal is to stay as close as possible to v4, just removing the padding would be an improvement

Reply:
UI update done